edelphi/src/main/java/fi/metatavu/edelphi/pages/IndexPageController.java

```diff
@@ -90,10 +103,33 @@ public int compare(DelfoiBulletin o1, DelfoiBulletin o2) {
         return o2.getCreated().compareTo(o1.getCreated());
       }
     });
+
+    if (!ignoreBulletinLocale) {
+      TextLanguageDetector langdetect = null;
+      try {
+        langdetect = TextLanguageDetector.getInstance();
+      } catch (IOException e) {
+        System.out.println("Failed to get TextLanguageDetector instance");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)